### PR TITLE
Html css hello microverse-project

### PR DESCRIPTION
**Pull request by ismayito**
In this project:
we used Html and CSS.
We used technologies such as Git, GitHub and linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1{
+    color:red;
+}
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Great job submitting your first Microverse project :clap:. As stated in the project requirements kindly make sure your `CSS` uses a `class` selector and not an `element selector 🙏 . Example:

```
.header {
  color: red
}
```

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @ismayito , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️

- [ ] Kindly ensure you set up the required linters as stated in the project requirement 🙏 . This can be done by following the steps provided by Microverse in this [article](https://github.com/microverseinc/linters-config/tree/master/html-css) ☺️ 
- [ ] Kindly make sure you have a descriptive `Readme`  that follows the template provided by [Microverse](https://github.com/microverseinc/readme-template).  🙏 
- [ ] Kindly make sure your pull request is descriptive and has a short summary of what you did in this project milestone. This can be done by following the steps provided by Microverse provided in this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/images/pull_request_desc.gif)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @ismayito ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] As stated by the previous reviewer, kindly add a readme file to your root directory and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template) for guidance.
- [ ] Kindly make sure your Linter checks are passing and are in green. This is a key part of the approval of your project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @ismayito 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**